style.css

```diff
@@ -238,6 +224,8 @@ body {
 }
 
 .title-1 {
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
```

Comment:
[Optional] Since your design uses `inter` globally, you could use the universal selector `*` to apply `font-family: 'Inter', Arial, Helvetica, sans-serif;` to all elements on the page. Be sure to check that everything still works and looks as directed by the figma template if you make this change